staff/dbenavente/SemanticMarks/leadtech-exercise/home.html

```diff
@@ -0,0 +1,234 @@
+                </li>
+                <li>
+                    <a href="https://leadtech.com/work-with-us">Work with us</a>
+                    <ul>
+                        <li>
+                            <a href="https://leadtech.com/work-with-us/backend">Backend</a>
+                        </li>
+                        <li>
+                            <a href="https://leadtech.com/work-with-us/bi">BI</a>
+                        </li>
+                        <li>
+                            <a href="https://leadtech.com/work-with-us/content-and-copywriting">Content and copywriting</a>
+                        </li>
+                        <li>
+                            <a href="https://leadtech.com/work-with-us/customer-service/">Customer Service</a>
+                        </li>
+                        <li>
+                            <a href="https://leadtech.com/work-with-us/finance">Finance</a>
+                        </li>
+                        <li>
+                            <a href="https://leadtech.com/work-with-us/frontend">Frontend</a>
+                        </li>
+                        <li>
+                            <a href="https://leadtech.com/work-with-us/human-resources">Human Resources</a>
+                        </li>
+                        <li>
+                            <a href="https://leadtech.com/work-with-us/project-manager">Project Management</a>
+                        </li>
+                        <li>
+                            <a href="https://leadtech.com/work-with-us/qa">QA</a>
+                        </li>
+                        <li>
+                            <a href="https://leadtech.com/work-with-us/sem">SEM</a>
+                        </li>
+                        <li>
+                            <a href="https://leadtech.com/work-with-us/seo">SEO</a>
+                        </li>
+                        <li>
+                            <a href="https://leadtech.com/work-with-us/social-media">Social Media</a>
+                        </li>
+                        <li>
+                            <a href="https://leadtech.com/work-with-us/sysadmin">Sysadmin</a>
+                        </li>
+                        <li>
+                            <a href="https://leadtech.com/work-with-us/ux">UX</a>
+                        </li>
+                    </ul>
+                </li>
+                <li>
+                    <a href="https://leadtech.com/entrepreneur">Entrepreneurs</a>
+                </li>
+                <li>
+                    <a href="https://leadtech.com/investor">Investors</a>
+                </li>
+                <li>
+                    <a href="https://leadtech.com/contact">Contact</a>
+                </li>
+                <li>
+                    <a href="https://leadtech.com/">Leadtech</a>
+                </li>
+            </ul>
+        </nav>
+    </header>
+    <section>
```

Comment:
This block could be wrapped in a `main` tag → https://developer.mozilla.org/en-US/docs/Web/HTML/Element/main